Authgear.Shared/AuthenticateOptions.cs

```diff
@@ -21,13 +21,13 @@ public AuthenticateOptions(string redirectUri)
         {
             RedirectUri = redirectUri;
         }
-        internal OidcAuthenticationRequest ToRequest(bool suppressIdpSessionCookie)
+        internal OidcAuthenticationRequest ToRequest(bool ssoEnabled)
```

Comment:
Nitpick: If there is no platform-alignment concern, the convention of bool in C# is `isSsoEnabled` @_@

Reply:
How about if we change all SDKs to use `isSSOEnabled`?
So for xamarin, it will be `isSsoEnabled`. And for the others, it will be `isSSOEnabled`. What do you think? @louischan-oursky

Reply:
I am fine with isSSOEnabled or isSsoEnabled

Reply:
Updated 🙏🏻